frontend/src/app/person-edit/person-edit.component.html

```diff
@@ -230,19 +230,21 @@ <h3>Situation fiscale</h3>
     <div class="form-group row" *ngIf="personForm.value.fiscalStatus !== 'UNKNOWN'">
       <label for="fiscalStatusDate" class="col-sm-3 col-form-label">Date de mise en place</label>
       <div class="col-sm-9 input-group">
-        <div class="input-group-addon" (click)="fsd.toggle()">
-          <i class="fa fa-calendar" aria-hidden="true"></i>
+        <div class="input-group-prepend" (click)="fsd.toggle()">
+          <span class="input-group-text fa fa-calendar" aria-hidden="true"></span>
         </div>
         <input class="form-control" id="fiscalStatusDate" placeholder="01/01/2016"
                formControlName="fiscalStatusDate" ngbDatepicker #fsd="ngbDatepicker" [minDate]="{year: 1990, month: 1}">
       </div>
     </div>
-    <div *ngIf="personForm.value.fiscalStatus !== 'UNKNOWN'" class="form-check row">
+    <div class="row">
       <div class="col-sm-9 offset-sm-3">
-        <label class="form-check-label">
-          <input type="checkbox" class="form-check-input" id="fiscalStatusUpToDate" formControlName="fiscalStatusUpToDate" />
-          A jour
-        </label>
+        <div *ngIf="personForm.value.fiscalStatus !== 'UNKNOWN'" class="form-check">
+          <label class="form-check-label">
```

Comment:
The `label` still wrap the `input` here: is it normal?

Reply:
Nope. Will fix. Good catch.